zcash_primitives/src/transaction/components/transparent.rs

```diff
@@ -112,18 +132,17 @@ impl OutPoint {
         writer.write_u32::<LittleEndian>(self.n)
     }
 
-    /// Returns `true` if this `OutPoint` is "null" in the Bitcoin sense: it points to the
-    /// `u32::MAX`th output of the transaction with the all-zeroes txid.
+    /// Returns `true` if this `OutPoint` is "null" in the Bitcoin sense.
     fn is_null(&self) -> bool {
```

Comment:
I went and looked and this is used exclusively for transparent coinbase identification. As such, the method is kind of badly named - I would prefer something like `is_coinbase_prevout` - we don't even need the null constant, because we never want to construct and use that value, just check against it to determine whether it corresponds to the pattern that is required for coinbase.